gateway/api/guardrails/guardrails.go

```
@@ -158,14 +229,18 @@ func Get(c *gin.Context) {
 	case models.ErrNotFound:
 		c.JSON(http.StatusNotFound, gin.H{"message": "resource not found"})
 	case nil:
+		// Get connections for this rule
+		connectionIDs, _ := models.GetConnectionIDsForGuardRail(ctx.GetOrgID(), rule.ID)
```

Don't hide the error, at least log to the stdout